crates/holochain_deterministic_integrity/src/lib.rs

```diff
@@ -1,4 +1,43 @@
-//! The Holochain Deterministic Integrity
+//! Holochain Deterministic Integrity (HDI) is Holochain's data model and integrity toolset for
+//! writing zomes.
+//!
+//! The logic of a Holochain DNA can be divided into two parts: integrity and coordination.
+//! Integrity is the part of the hApp that defines the data structure and validates data
+//! types and values. Coordination encompasses the domain logic and implements the functions
```

Comment:
```suggestion
//! types. Coordination encompasses the domain logic and implements the functions
```
(to go along with my previous suggestion -- wish I could do multi-line suggestions)